src/libraries/reduxUtils/convert.ts

```diff
@@ -7,7 +7,9 @@ export const convertToSummaryData = (
 ) => {
   const res = data.map(({ ...rest }) => ({
     type: field.type,
-    date: moment(rest[field.dateField]).format("DD/MM/YYYY"),
+    date: moment(rest[field.dateField]).isValid()
+      ? moment(rest[field.dateField]).format("DD/MM/YYYY")
+      : moment(+rest[field.dateField]).format("DD/MM/YYYY"),
```

Comment:
Are you trying to format a date even if it's invalid?

Reply:
We have two cases: millisecondes and date string. invalid date is also possible.

Reply:
OK, I changed it